.travis.yml

```diff
@@ -8,7 +8,7 @@ env:
         - secure: "QFiZt1xlZujuXksbCQPc0zFdGFBzuCDyXVi2kOjAbwzhHzwp4dOOXDlJbZAvRPCEeQ6Z2wj9iAAtHWWY9kL0au8vnqCzDWRj8ep99ckBPZVVNW/pEG0VC64MxftD+tscEuUSFrCd4GDyB+0YKAYSkRs4Ww766enyz4pPS7IbQ9Jj40bCHKAPaBhbH09bORDHVYzkcCrlNTv/qRClMsoKWZMLVW2nnOda2nF4/fJkgG89/G6keCUO514IwAmZiy0r9Swe5pDRmzaSIxddy86d2tLayoiX/iOKYTBRvv3NtM0xDOkVARsVUnw3MTWTIuWical0F5KhhZuyo5sRoXs1q/BpmD1YTTzOavtsMpXXj5HJwLBiVU4KdElL1olAwfRZMnJhhPId97qHMDvzu+ikscRKSlfghnTZaCpjiUx9wW69KnwDEpe7uOh4KodcQfYLkYnNzx6qKEpx/9w0StzKS03iTaYu1zoNmu4bVJFtHcNTmUVZUHRuiU9EBTWdOBtSlKDKA8zYa5i3uqhZrflAcvRG2siPXoE5UcFPQQE0J0hoE6o8NmZe4EgDyYJcg0gcJp3SX1LLNNcHa0d1P+zuKVL0NNEFmWn/oPbtaUQHKFVE4nC375yqjcaH2RCGCGQs3j5tk0s7ZmsbxnZ1D1Xz5uj6IVeymruhU7pCKlNc6cc="
 
 node_js:
-  - 8.9.0
+  - 9.10.0
```

Comment:
I'd go with 10.x LTS

Reply:
I can a have look if it breaks sth or not.